src/rules/no-large-snapshots.ts

```diff
@@ -78,28 +103,34 @@ export default {
       },
     ],
   },
-  create(context) {
+  defaultOptions: [{}],
+  create(context, [options]) {
     if (context.getFilename().endsWith('.snap')) {
       return {
         ExpressionStatement(node) {
-          reportOnViolation(context, node);
+          reportOnViolation(context, node, options);
         },
       };
     } else if (context.getFilename().endsWith('.js')) {
       return {
         CallExpression(node) {
-          const propertyName =
-            node.callee.property && node.callee.property.name;
           if (
-            propertyName === 'toMatchInlineSnapshot' ||
-            propertyName === 'toThrowErrorMatchingInlineSnapshot'
+            'property' in node.callee &&
```

Comment:
@SimenB I know what you're going to say - I'm going to implement varargs in it's own PR, as it's a waterfall change that while relatively straightforward, does require a bit of its own work due to subtyping.

The main reason I replaced the array here is this way `isSupportedAccessor` has full coverage; otherwise I'd have left it as `includes` & done the varargs thing in another PR.